src/main/proto/netflix/titus/validator/validator.proto

```diff
@@ -226,3 +230,28 @@ message ImageValidationResponse {
 
     CacheMetadata cacheMetadata = 3;
 }
+
+message NetworkParametersValidationRequest {
+
+    // Account Id network address allocation is being requested in.
+    string accountId = 1;
+
+    // Subnets for network address allocation.
+    repeated string subnets = 2;
+
+    // SecurityGroups to check for existence.
+    repeated string securityGroups = 3;
+}
+
+message NetworkParametersValidationResponse {
+    message Success {
+    }
+
+    oneof Result {
+        Success success = 1;
+
+        ValidationFailures failures = 2;
+    }
+
+    CacheMetadata cacheMetadata = 3;
```

Comment:
This structure is used to store last validation result for a configured TTL to prevent expensive rpc calls.

Reply:
What's the expected semantic to be?

Reply:
https://github.com/Netflix/titus-api-definitions/blob/master/src/main/proto/netflix/titus/validator/validator.proto#L43

To forward the context to the caller if the ValidationService API if the response served was cached. 
This would  cause less calls from ValidationService to VPC Service.

Reply:
Please document the semantics, or link to the semantics or leave a comment.